low-level/sensors/impl/generic/node_sas_port.py

```diff
@@ -419,16 +419,14 @@ def _create_json_message(self, alert_type, port):
 
         specific_info = {}
         specific_info_list = []
-
+        description = "N/A"
         if port != -1:
             # This is a port level alert, add an error key in specific info
             if alert_type == 'fault':
                 description = f"No connectivity detected on the SAS port {port}, possible \
 causes could be missing SAS cable, bad cable connection, faulty cable or SAS port failure"
             elif alert_type == 'fault_resolved':
                 description = "Connection established on SAS port."
-            specific_info_list.append(specific_info)
```

Comment:
why removing specific info

Reply:
This is not needed, before ,  instead of description here we were defining "event" string that we add  in specific_info. 
But as we changed event to description and description we added in "info", so this is not needed.

Reply:
Only event was there in specific_info? If yes, then i understand

Reply:
resource_id and "negotiated_link_rate are also there in specific info, these keys we are appending to specefic_info on line: 442.443.
 
 on line 430 we were adding "event" into specific_info, only that part  I have removed.

Reply:
understood.